### PR TITLE
Add score detail page

### PR DESCRIPTION
**Pull request by nanaya**
- not sure if the rank query is fast enough (my db still hasn't been updated with correct index)
- the accuracy dial grade points aren't quite correct
  - especially because there's no direct relation between accuracy and grade
- still uses current (older design) user card
- only for high scores at the moment (`osu_scores_*high`)
  - the url scheme will need some rethinking to handle other scores; this one currently follows replay download url

~~Missed some stuff (should only link best scores), waiting for #5960 before fixing.~~ Updated.

### CODE REVIEW

**Comment by peppy** [comment]
Is this okay to review now?

**Comment by nanaya** [comment]
yes

File: resources/assets/lib/play-detail-menu.tsx [hunk 1]
```diff
@@ -19,6 +19,13 @@ export class PlayDetailMenu extends React.PureComponent<Props> {
 
     const children = (dismiss: () => void) => (
       <>
+        <a
+          href={route('scores.show', { mode: score.mode, score: score.best_id })}
```
[the remark this answers is not in the document]
Comment (nanaya) [reply resources/assets/lib/play-detail-menu.tsx:23]: Hmmm I added `hasShow` check but typescript can't infer that `best_id` is not null after the check.

**Comment by nanaya** [comment]
Seems fine in my chrome ?_? 81.0.4044.138

**Comment by notbakaneko** [comment]
Is this some mac thing again, except this time it's Chrome instead of Safari ಠ_ಠ

**Pull request merged**